security-openid-connect-quickstart/src/test/java/org/acme/security/openid/connect/UsersResourceTest.java

```diff
@@ -0,0 +1,43 @@
+package org.acme.security.openid.connect;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.junit.jupiter.api.Test;
+
+@QuarkusTest
+@QuarkusTestResource(KeycloakServer.class)
+public class UsersResourceTest {
```

Comment:
@cemnura Can you please have a single test class (say `BearerTokenAuthenticationTest`) combining both admin and user resources tests - to have KC started and stopped once and avoid some instability.

Reply:
@sberyozkin gladly. Resolving ASAP.

Reply:
I combined the test classes into one class being `BearerTokenAuthenticationTest`.